pyproject.toml

```
@@ -91,7 +91,7 @@ all_extras = [
   'hmmlearn>=0.2.7; python_version < "3.11"',
   "holidays",
   'keras-self-attention; python_version < "3.11"',
-  "matplotlib>=3.3.2",
+  "matplotlib<3.9.1,>=3.3.2",
```

```suggestion
  "matplotlib!=3.9.1,>=3.3.2",
```

It is better to specifically exclude the broken version than to put a cap.